hudi-spark-datasource/hudi-spark-common/src/main/scala/org/apache/hudi/HoodieCreateRecordUtils.scala

```diff
@@ -194,7 +192,7 @@ object HoodieCreateRecordUtils {
     true
   }
 
-  private def validateMetaFieldsInAvroRecords(avroRec: GenericRecord): Unit = {
+  def validateMetaFieldsInAvroRecords(avroRec: GenericRecord): Unit = {
```

Comment:
why removed private access?

Reply:
For DELETE, this function is being called by `HoodieSparkSqlWriter.scala`.